tools/accuracy_checker/accuracy_checker/preprocessor/README.md

```diff
@@ -121,3 +121,4 @@ Accuracy Checker supports following set of preprocessors:
 * `clip_audio` - slices audio into several parts with equal duration
   * `duration` - sets duration in seconds
   * `max_clips` - sets the maximum number of clips (by default `1`)
+* `audio_normalization` - normalize audio record with mean sample subtraction and division on standard deviation of samples.
```

Comment:
What do you think about renaming to `normalize_audio` for conformity with others audio preprocessors? (Or even rename *them* to `audio_smth`?)

Reply:
еI thought about it and initially it has such name, but all other normalization functions has `normalization` in name. So I decide that it will be better to align name with other preprocessing for this block

Reply:
@eizamaliev , I have question about slide window, Could I add it here or you already has been worked on it?

Reply:
Almost ready, I'm testing it now